Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14.16.1
+FROM node:16.20.2-bookworm-slim
```

Comment:
why this custom image `bookworm-slim` instead of plain 16.20.2 ?

Reply:
have we compared the vulnerabilities that are getting fixed in `16.20.2` vs `16.20.2-bookworm-slim` ?

Reply:
**node:<version>-slim**
This image does not contain the common packages contained in the default tag and only contains the minimal packages needed to run node. Unless you are working in an environment where only the node image will be deployed and you have space constraints, we highly recommend using the default image of this repository.
Ref: https://hub.docker.com/_/node/

Reply:
we can also explore updating to latest node 20.x version or at least 18.x ?

Reply:
Thank you for pointing it out. This is an automated PR created by Snyk based on the specific version `node:16.20.2-bookworm-slim` .